wp-content/themes/core/assets/pcss/typography/anchors.pcss

```diff
@@ -6,6 +6,11 @@
 
 a {
```

Comment:
❓ Do we need some sort of front end selector for this so that these styles don't push out to other things? I think Dave was talking about this, if you solved this somewhere else feel free to ignore!

Reply:
Yep, great call out. We solved for this by splitting out an `editor.pcss` file in the `/pcss/typography` directory. So, this partial isn't loaded in the editor at all.